themes/default/content/challenge/holiday-shopping.md

```diff
@@ -0,0 +1,268 @@
+---
+title: "Holiday Shopping"
+layout: challenge/single
+description: |
+    Create a simple shopping list app using AWS Fargate and MongoDB Atlas
+meta_desc: |
+    Create a simple shopping list app using AWS Fargate and MongoDB Atlas
+meta_image: /images/challenge/challenge-dec-meta.png
+banner_image: /images/challenge/challenge-dec-banner.png
+---
+
+## Welcome to the Pulumi Challenge!
+
+### Holiday Shopping List
+
+The holidays are upon us and mere anarchy has been loosed upon our psyches. More specifically how do you keep track of everything you need to buy for gifts or food?!? Surely some salvation is at hand? Lucky for you, our friend, The Pulumipus, has created a simple to-do list application to bring order to your life. In this challenge, you will build this simple to-do list application that has a React frontend, Express backend, and a MongoDB database. This frontend and backend of the app run in Fargate with an Application Load Balancer (ALB) in front and the database is running in a MongoDB Atlas cluster.
+
+![alt_text](/images/challenge/architecture_mern.png "mern arch")
+
+If you feel so inclined to thank The Pulumipus, please write a blog post or post a quick video about it. Tag us on social media or email us at [da@pulumi.com](mailto:da@pulumi.com), and we will send you some special swag.
+
+### Prerequisites
+
+In order to complete this challenge, you'll need a couple things set up in advance. The entire challenge can be run on the free tiers of AWS and MongoDB Cloud.
+
+- A [Pulumi account](https://app.pulumi.com/signup)
+- The [Pulumi CLI](/docs/get-started/install/)
+- [Python 3.9 or higher](https://www.python.org/downloads/)
+- Docker
+- An [AWS account](https://portal.aws.amazon.com/)
+- The [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
+- A [MongoDB Cloud account](https://www.mongodb.com/cloud/atlas/signup)
+- A [GitHub account](https://github.com/signup)
+
+### Challenge
+
+#### Step 1. Initialize a project
+
+You will first clone the base application and then the rest of the steps will take you through building the Pulumi program.
+
+```shell
+git clone -b challenge https://github.com/aaronkao/atlas-fargate.git
+```
+
+Next, initialize the stack.
+
+```shell
+cd atlas-fargate
+pulumi stack init dev
+```
+
+You will also need to export your AWS and MongoDB Cloud credentials and your AWS region (e.g., us-west-2). To access your MongoDB Cloud credentials, go to the __Organization Access Manager__ panel and the __API Keys__ tab.
```

Comment:
We need to tell them what to pick here, in case they're creating a new key (which they probably will be):

![image](https://user-images.githubusercontent.com/274700/205185275-7f8d83d5-38b3-4c40-b4d5-c6523d7e9338.png)

I believe you need at least Project Creator, as the others all appear to be read-only.